config/jobs/kubernetes/sig-autoscaling/azure/cluster-autoscaler-azure-presubmits.yaml

```diff
@@ -45,6 +45,8 @@ presubmits:
             value: v1.29.4
           - name: CLUSTER_TEMPLATE
             value: ${GOPATH}/src/k8s.io/autoscaler/cluster-autoscaler/cloudprovider/azure/test/templates/cluster-template-prow-aks-aso-cluster-autoscaler.yaml
+          - name: CLUSTER_CREATE_ATTEMPTS
```

Comment:
Is this an existing ci-entrypoint.sh variable?

Reply:
Yes:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/facb71fa25c1f06247e14f4ab10cf55b629e8a4d/hack/create-dev-cluster.sh#L90